core/src/test/scala/integration/kafka/tools/DumpLogSegmentsIntegrationTest.scala

```diff
@@ -0,0 +1,66 @@
+package integration.kafka.tools
+
+import java.io.{ByteArrayOutputStream, PrintStream}
+import java.util.Properties
+
+import kafka.integration.KafkaServerTestHarness
+import kafka.log.Log
+import kafka.server.KafkaConfig
+import kafka.tools.DumpLogSegments
+import kafka.utils.TestUtils
+import org.apache.kafka.clients.producer.{KafkaProducer, ProducerConfig, ProducerRecord}
+import org.apache.kafka.common.serialization.ByteArraySerializer
+import org.junit.Assert.assertTrue
+import org.junit.Test
+
+class DumpLogSegmentsIntegrationTest extends KafkaServerTestHarness {
```

Comment:
A brief description on what is tested would be nice.

Reply:
Just done